src/v2i-hub/PreemptionPlugin/src/PreemptionPlugin.cpp

```diff
@@ -110,7 +110,7 @@ void PreemptionPlugin::HandleBasicSafetyMessage(BsmMessage &msg, routeable_messa
 	//if( it != allowedList.end())
 	//{	
 		if (bsm->partII != NULL) {
-			if (bsm->partII[0].list.count >= partII_Value_PR_SpecialVehicleExtensions ) {
+			if (bsm->partII[0].list.count >= PartIIcontent_126P0__partII_Value_PR_SpecialVehicleExtensions ) {
```

Comment:
Please check with SPAT plugin, preemption plugin  make sure they are not affected by this change.

Reply:
How would I check this. If I can compile and enable each without issue is that sufficient.